core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -989,6 +989,18 @@ public function checkRootType($varValue, Contao\DataContainer $dc)
 		return $varValue;
 	}
 
+	/**
+	 * Return the SERP URL
+	 *
+	 * @param Contao\PageModel $model
+	 *
+	 * @return string
+	 */
+	public function getSerpUrl(Contao\PageModel $model)
+	{
+		return $model->getAbsoluteUrl();
```

Comment:
@leofeyer If the model has no ID and no alias this call will throw an exception. This happens for example when you create a new page in the backend, this currently results in `Parameter "alias" for route "contao_frontend" must match ".+" ("" given) to generate a corresponding URL.`

Reply:
@aschempp has fixed this in #1169.